dev/bench/bench.sh

```diff
@@ -368,8 +376,23 @@ create_opam() {
     opam repo add -q --this-switch coq-extra-dev "$OPAM_COQ_DIR/extra-dev"
     opam repo add -q --this-switch coq-released "$OPAM_COQ_DIR/released"
 
-    # Pinning for packages that are not in a repository
-    opam pin add -ynq coq-perennial.dev git+https://github.com/mit-pdos/perennial#coq/tested
```

Comment:
Do you intend to move it somewhere else?

Reply:
We haven't benched perennial for a while so this is just a in-passing cleanup